lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -438,11 +437,7 @@ def configure_puppet_on(hosts, opts = {})
         # @api private
         def install_puppet_from_rpm_on( hosts, opts )
           block_on hosts do |host|
-            if host[:type] == 'aio'
-              install_puppetlabs_release_repo(host,'pc1',opts)
-            else
-              install_puppetlabs_release_repo(host,nil,opts)
-            end
+            install_puppetlabs_release_repo(host,opts[:puppet_collection],opts)
```

Comment:
@kevpl is this reasonable to have folks pass in `puppet_collection` to opts if they want to use something other than the default repo?

Reply:
it seems that they'll have to pass something in to use anything other than the default repos. Are you thinking that the name should be different, that they shouldn't be able to specify something other than the default repo, or something else?